CHANGELOG.md

```diff
@@ -50,18 +55,12 @@ release.
 - Fixed hardcoded file naming in the hijitter app dealing with output from pipeline. [#4372](https://github.com/USGS-Astrogeology/ISIS3/pull/4372)
 - Fixed "About Qview" to point to website documentation. [4333](https://github.com/USGS-Astrogeology/ISIS3/issues/4333)
 - Fixed bug where the time bias was not being added to the ephemeris times in ckwriter. [4129](https://github.com/USGS-Astrogeology/ISIS3/issues/4129)
```

Comment:
Should this be removed from the changelog since this change was reverted from dev?

Reply:
The PR for the revert was closed rather than merged, thus the change is still present in dev